ui/raidboss/data/07-dt/alliance/jeuno-first-walk.ts

```diff
@@ -0,0 +1,792 @@
+        return output.avoidCleave!();
+      },
+      run: (data) => data.splitterTargets = [],
+      outputStrings: {
+        cleaveOnYou: Outputs.tankCleaveOnYou,
+        avoidCleave: Outputs.avoidTankCleaves,
+      },
+    },
+    {
+      id: 'Jeuno First Walk Angels Gekko',
+      type: 'StartsUsing',
+      netRegex: { id: 'A07A', source: 'Ark Angel GK', capture: true },
+      durationSeconds: (_data, matches) => parseFloat(matches.castTime),
+      response: Responses.lookAwayFromSource(),
+    },
+    {
+      id: 'Jeuno First Walk Angels Meteor',
+      type: 'StartsUsing',
+      netRegex: { id: 'A08A', source: 'Ark Angel TT', capture: true },
+      durationSeconds: 5,
+      response: Responses.interruptIfPossible('alarm'),
+    },
+    {
+      id: 'Jeuno First Walk Spiral Finish',
+      type: 'StartsUsing',
+      netRegex: { id: 'A06C', source: 'Ark Angel MR', capture: false },
+      delaySeconds: 5.5,
+      response: Responses.knockback(),
+    },
+    {
+      id: 'Jeuno First Walk Angels Havoc Spiral Clockwise',
+      type: 'HeadMarker',
+      netRegex: { id: '00A7', capture: false },
+      durationSeconds: 5,
+      infoText: (_data, _matches, output) => output.clockwise!(),
+      outputStrings: {
+        clockwise: Outputs.clockwise,
+      },
+    },
+    {
+      id: 'Jeuno First Walk Angels Havoc Spiral Counterclockwise',
+      type: 'HeadMarker',
+      netRegex: { id: '00A8', capture: false },
+      durationSeconds: 5,
+      infoText: (_data, _matches, output) => output.counterclockwise!(),
+      outputStrings: {
+        counterclockwise: Outputs.counterclockwise,
+      },
+    },
+    {
+      id: 'Jeuno First Walk Angels Dragonfall',
+      type: 'StartsUsing',
+      netRegex: { id: 'A07E', source: 'Ark Angel GK', capture: false },
+      alertText: (_data, _matches, output) => output.stacks!(),
+      outputStrings: {
+        stacks: Outputs.stacks,
+      },
+    },
+    {
+      id: 'Jeuno First Walk Angels Arrogance Incarnate',
+      type: 'HeadMarker',
+      netRegex: { id: '0131', capture: true },
+      response: Responses.stackMarkerOn(),
+    },
+    {
+      id: 'Jeuno First Walk Angels Guillotine',
+      type: 'StartsUsing',
+      netRegex: { id: 'A067', source: 'Ark Angel TT', capture: false },
+      durationSeconds: 10,
+      response: Responses.getBehind(),
+    },
+    {
+      id: 'Jeuno First Walk Angels Dominion Slash',
+      type: 'StartsUsing',
+      netRegex: { id: 'A085', source: 'Ark Angel EV', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Jeuno First Walk Angels Holy',
+      type: 'StartsUsing',
+      netRegex: { id: 'A089', source: 'Ark Angel EV', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Jeuno First Walk Angels Proud Palisade',
+      type: 'AddedCombatant',
+      netRegex: { npcBaseId: '18260', capture: false }, // Ark Shield
+      response: Responses.killAdds(),
```

Comment:
Suggest something along the lines of `Break shield` here, since the other add is targetable while the shield makes it immune.